src/EnergyPlus/ChillerReformulatedEIR.cc

```diff
@@ -1457,19 +1456,8 @@ namespace ChillerReformulatedEIR {
                     ShowContinueError(
                         "EIR as a function of PLR curve output at various part-load ratios and condenser water temperatures shown below:");
                     ShowContinueError("PLR           =    0.00   0.10   0.20   0.30   0.40   0.50   0.60   0.70   0.80   0.90   1.00");
-                    std::string StringVar; // Used for EIRFPLR warning messages
-                    ObjexxFCL::gio::write(StringVar, "'Cond Temp(C) = '");
-                    for (int CurveValPtr = 1; CurveValPtr <= 11; ++CurveValPtr) {
-                        ObjexxFCL::gio::write(StringVar, "(F7.2,$)") << CondTempArray(CurveValPtr);
-                    }
-                    ObjexxFCL::gio::write(StringVar);
-                    ShowContinueError(StringVar);
-                    ObjexxFCL::gio::write(StringVar, "'Curve Output = '");
-                    for (int CurveValPtr = 1; CurveValPtr <= 11; ++CurveValPtr) {
-                        ObjexxFCL::gio::write(StringVar, "(F7.2,$)") << CurveValArray(CurveValPtr);
-                    }
-                    ObjexxFCL::gio::write(StringVar);
```

Comment:
This is definitely a bug on my part. I missed this. It needs to be written to stdout. @Myoldmopar I'll address this before I take the PR out of draft mode.

Reply:
@lefticus @Myoldmopar No, this isn't writing to stdout. This is writing formatted values to `StringVar` and then `StringVar` is the error string in line 1472 `ShowContinueError(StringVar);`. So, I think you've covered this with the new lines 1459 and 1460.  I doubt this code ever gets exercised in the standard test suite, we need to create a file that causes this error to fire to see if the new code is working.